README.md

```diff
@@ -60,26 +60,26 @@ package main
 import "github.com/alecthomas/kong"
 
 var CLI struct {
-  Rm struct {
```

Comment:
Can you revert these whitespace changes please?

Reply:
Yes, looks like my IDE was being too helpful :/